packages/semantic-tokens/scripts/legacyTokens.ts

```diff
@@ -0,0 +1,49 @@
+import fs from 'node:fs';
+import path from 'node:path';
+
+// eslint-disable-next-line no-restricted-imports
+import * as tokensPackage from '@fluentui/tokens';
+import { fluentOverrides } from './fluentOverrides';
+
+main();
+
+function main() {
+  console.log('Importing required fluent legacy tokens as flat export');
+
+  const semanticTokenFallbacks = Object.keys(fluentOverrides);
+  const fluentTokens = Object.keys(tokensPackage.tokens);
+  const comment = '// THIS FILE IS GENERATED AS PART OF THE BUILD PROCESS. DO NOT MANUALLY MODIFY THIS FILE\n';
+
+  const generatedTokens = semanticTokenFallbacks.reduce((acc, t) => {
+    const fluent2Fallback = fluentOverrides[t].f2Token;
+    if (!fluent2Fallback) {
+      return '';
+    }
+    if (!fluentTokens.includes(fluent2Fallback)) {
+      // This should never occur, but let's flag if a mistake was made in fallback token names
+      throw new Error(`Fluent token ${fluentOverrides[t].f2Token} not found in fluent tokens`);
+    }
+
+    const token = `
+/**
```

Comment:
Nit: Might be prettier, but the indenting is a bit funky to read here. If we can fix that, that'd be nice.